### PR TITLE
Add link to disabling accidental OTP triggers

### PR DESCRIPTION
**Pull request by cainejette**
This hits many of my coworkers and I think linking the instructions to disabling this behavior would go a long way to reducing this.

### CODE REVIEW

File: src/yubikey-invalidation.coffee [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = (robot) ->
       if res.statusCode != 200
         msg.reply "#{messagePrefix} I tried to invalidate that OTP for you, but I got a #{res.statusCode} error from the server 😢"
       else
-        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒"
+        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒 You can disable accidental OTP triggers via https://support.yubico.com/hc/en-us/articles/360013714379-Accidentally-Triggering-OTP-Codes-with-Your-Nano-YubiKey"
```
Comment (chesterbr) [inline src/yubikey-invalidation.coffee:36]: Minor tweak; feel free to ignore (from [prior art](https://github.com/hubot-scripts/hubot-gh-issues/blob/719e93350eae312d111ccf545e4cd945d70e76f7/index.coffee#L20)):

```suggestion
        msg.reply "#{messagePrefix} I went ahead and invalidated that OTP for you 🔒\nYou can disable accidental OTP triggers via https://support.yubico.com/hc/en-us/articles/360013714379-Accidentally-Triggering-OTP-Codes-with-Your-Nano-YubiKey"

```

**Review by chesterbr — APPROVED** [review]
LGTM, thanks! Comment is non-blocker.

**Comment by chesterbr** [comment]
Also: it may require a version bump (see https://github.com/hubot-scripts/hubot-yubikey-invalidation/commit/c9c1ae5a6212e2dcf59290dc3bc0b54f4c395b12). Not sure how project maintainers are doing this though; didn't find docs on that (but I may have missed 😅)

**Comment by fine-fiddle** [comment]
I think we should use the [old instructions we put on the wiki](https://engineering.paypalcorp.com/confluence/display/ITKB/Reconfiguring+your+Yubikey) rather than Yubico's instructions

Yubico suggests disabling YOTP entirely as their first suggestion, and that'll break the yubikey for PingID. Most of the people who read the yubico instructions won't be technical enough to understand that at first, even if they're capable of following the instructions by rote.

**Pull request closed without merging**